src/settings.js

```diff
@@ -737,18 +737,10 @@ var WASM = 1; // Whether to use compile code to WebAssembly. Set this to 0 to co
               // of the port, which can useful for local dev work on binaryen itself).
 
 var WASM_BACKEND = 0; // Whether to use the WebAssembly backend that is in development in LLVM.
-                      // This requires that BINARYEN be set, as we use Binaryen's s2wasm to
-                      // translate the backend output.
+                      // This requires that BINARYEN be set, as we use Binaryen's wasm-emscripten-finalize to
```

Comment:
I'd change it to "this requires WASM be set", because we changed the canonical name of the flag